unity-renderer/Assets/DCLPlugins/LoadingScreenPlugin/LoadingScreenPlugin.cs

```diff
@@ -45,7 +43,7 @@ public void Dispose()
             loadingScreenController.Dispose();
         }
 
-        public static async UniTask<LoadingScreenView> CreateLoadingScreenView(IAddressableResourceProvider assetsProvider, CancellationToken cancellationToken) =>
-            await assetsProvider.Instantiate<LoadingScreenView>(LOADING_SCREEN_ASSET, cancellationToken: cancellationToken);
+        public static LoadingScreenView CreateLoadingScreenView() =>
+            GameObject.Instantiate(Resources.Load<GameObject>(LOADING_SCREEN_ASSET)).GetComponent<LoadingScreenView>();
```

Comment:
Just want to make sure: why we switched back to `Resources`?

Reply:
Artifacts were discovered while using the LoadingScreen in this [issue](https://github.com/decentraland/unity-renderer/issues/4460). Looks more related to the loading screen not being available from start rather than the async delay caused by addressables.

Nonetheless, I think it makes sense to add the LoadingScreen to the main build. It's the first thing that should be shown without any delay, and having an async initialization may cause artifacts.